packages/vscode-graphql/package.json

```
@@ -179,7 +179,7 @@
     "ovsx": "0.8.3"
   },
   "dependencies": {
-    "graphql": "^16.8.1",
+    "graphql": "^16.8.1 || ^17.0.0-alpha.2",
```

```suggestion
    "graphql": "^16.8.1 || ^17.0.0-alpha.5",
```
better to points on the working version?

---

yeah, alpha.2 should work as well, it just has the old duplicating format, which was introduced in alpha.3

alpha.5 (and probably alpha.4) currently has a bug with defer nested under stream